src/scss/theme.scss

```diff
@@ -0,0 +1,34 @@
+// scss-custom theme-color-variables
+$primary: #60b15e;
+$warning: #40701a60;
+$list-group-action-hover-color: #fff;
+$list-group-hover-bg: #009be4;
+$list-group-bg: #d3d3d3;
+$list-group-border-color: none;
+//Text
+$link-decoration: none;
+
+//Custom css
```

Comment:
Hi Geir.  I visited your site via Netlify and things were looking, everythings looks great, I visited your site via Netlify and things were looking great! I noticed that you have created custom css classes here, is it working correctly with scss? I have written my custom classes differently. This is an example of one of my variables: // Example
$shadow1: 2px;
$shadow2: 12px;
$shadow3: 4.4px;
$shadow4: 0px;
$color: #332348;

.box-shadow {
  box-shadow: $shadow1 $shadow2 $shadow3 $shadow4 $color;
}.
Your way may be correct as well! Great work!